core/src/main/java/com/linecorp/armeria/common/HttpHeadersBase.java

```diff
@@ -55,6 +62,9 @@
     private static final String[] PROHIBITED_VALUE_CHAR_NAMES;
     private static final char LAST_PROHIBITED_VALUE_CHAR;
 
+    static final List<LanguageRange> ACCEPT_LANGUAGE_WILDCARD =
+            ImmutableList.copyOf(LanguageRange.parse("*"));
+
```

Comment:
Should this be exposed somewhere?

Reply:
> I cannot find any LanguageRange with "*", and the `sun.util.Locale.LocaleMatcher` that contains e.g. the actual implementation of the LanguageRange.parse() does not use a constant.
> 
> I can add that class, but it will be for just the single LanguageRange, as the problem is that a `LanguageRange` contains both the `Locale` and a weight. It might be that there are certain combinations of `Locale` and weight that are common, but I believe they would be very localized to certain countries. It may be that even USA and UK does not share any `LanguageRange`s at all.
> 
> I have no opinion on where to put that constant, so what do you want? Btw, if the `acceptLanguages()`-method is made to return `null` instead of accept all, I don't see a need for Armeria to define this constant.

Removed the `LanguageRange` constant completely as I'm returning null from acceptLanguages() instead. After thinking about it again, that was the better solution imho.